azure/template.json

```diff
@@ -38,7 +38,8 @@
         "stubFunctionAppName": "[concat(variables('resourcePrefix'),'-stub-fa')]",
         "functionAppName": "[concat(variables('resourcePrefix'),'-data-fa')]",
         "importFunctionAppName": "[concat(variables('resourcePrefix'),'-import-fa')]",
-        "storageAccountName": "[concat('das',toLower(parameters('resourceEnvironmentName')),parameters('serviceName'),'str')]"
+        "storageAccountName": "[concat('das',toLower(parameters('resourceEnvironmentName')),parameters('serviceName'),'str')]",
+        "storageContainerNames": [ "import-person", "import-campaign-members" ]
```

Comment:
Thoughts on having the storageContainerNames as a parameter? Gives the arm template a little flexibility if we're ever adding more containers possibly?

Reply:
I did consider that but my expectation is that they'll be few and far between and any additional containers would be accompanied by code changes so coding them into the ARM template makes sense.  Happy to change if the standard pattern is to pass them in as a param though

Reply:
Most apps create the containers via code and any that do via arm template has been a mix of arm template as well as parameters passed in 👍